docs/_config.yml

```diff
@@ -280,12 +280,23 @@ es:
       sections:
         - path: es/week08/08-1.md
         - path: es/week08/08-2.md
+    - path: es/week09/09.md
+      sections:
+        - path: es/week09/09-1.md
+        - path: es/week09/09-2.md
+        - path: es/week09/09-3.md
```

Comment:
We don't have the translation of week 9. Why did you add it?

Reply:
I was just thinking of adding scaffolding, but you're right, it's likely to cause merge conflicts, so I removed it.